stl/inc/xstring

```diff
@@ -1983,19 +1983,19 @@ public:
     }
 
 #if _ITERATOR_DEBUG_LEVEL >= 1
-    friend void _Verify_range(const _String_const_iterator& _First, const _String_const_iterator& _Last) {
+    friend void _Verify_range(const _String_const_iterator& _First, const _String_const_iterator& _Last) noexcept {
         _STL_VERIFY(_First._Getcont() == _Last._Getcont(), "string iterators in range are from different containers");
         _STL_VERIFY(_First._Ptr <= _Last._Ptr, "string iterator range transposed");
     }
 #endif // _ITERATOR_DEBUG_LEVEL >= 1
 
     using _Prevent_inheriting_unwrap = _String_const_iterator;
 
-    _NODISCARD const value_type* _Unwrapped() const {
+    _NODISCARD const value_type* _Unwrapped() const noexcept {
         return _Unfancy(_Ptr);
     }
 
-    void _Seek_to(const value_type* _It) {
+    void _Seek_to(const value_type* _It) noexcept {
         _Ptr = _Refancy<pointer>(const_cast<value_type*>(_It));
     }
 

```

Comment:
Non-member `operator+` below isn't being marked `noexcept`, but for the other random-access iterators it is.